REDHOUSEVE/edx-platform
# integrate sketchfab viewer tab

**Description:** 
Integrate sketchfeb viewer as a course tab.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-1035

**Testing instructions:**

1. Create a new course or use an existing one
2. Goto advanced settings
3. Change anything (and revert back if no change is required)
4. Save settings
5. Go to course in LMS
6. "Sketchfab viewer" tab should be enabled. Can be hidden from Studio "pages" page.
**Merge checklist:**


**Post merge:**


**Note:** Please add screenshots for any viewable change.
